wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -100,6 +100,15 @@ function arcsiStop(episodeName) {
     });
 }
 
+function arcsiDownload(episodeName, downloadLink) {
+    gtag('event', 'Arcsi download', {
+        'event_category': episodeName,
+        'event_label': 'Download name',
+        'value': 1,
+    });
```

Comment:
Why don't make it consistent with the `Radio play` event (category = show title)? See: https://github.com/mmmnmnm/lahmacun/blob/1d04ecd14873e7b5896419c778d157af08cd5154/wp/wp-content/themes/lahma_maker/assets/playerfiles/radio_player.vue#L326-L330

Also, we could use the `event_label` for the episode name, don't we? Not only here, for radio play events too (we never make sure of `Play state`).

Reply:
?? don't understand the comment. the download title is the same as the play title, we already agreed upon last time. you sent the radio stream title, which we get from different source
in content-arcsi.php
```
$fullTitle = $showtitle . ' | ' . $showname;
title="<?php echo $fullTitle; ?>
```